File: calc.js
let nums = [77,12345,4,0,8,7];
const _ = require('lodash');
const {
    double_each,
    square_each,
    sum,
    mean,
    sort,
    median
} = require('./calc_functions');


if(_.isArray(nums)){
    console.log('numbers:', nums);
    console.log('sorted:', sort(nums));
    console.log('double:', double_each(nums));
    console.log('square:', square_each(nums));
    console.log('sum:', sum(nums));
    console.log('mean:', mean(nums));
    console.log('median:', median(nums));
}

File: calc.java
public class calc {
  public static void main(String[] args) {
    System.out.println("Hello World");
    int[] numbers = {1,2,3,4,5,6};
    System.out.println(numbers);
    System.out.println("sum:" + sum(numbers));
  }

  public static int sum(int[] nums){
    int s = 0;
    for(int i : nums)
      s+=i;
    return s;
  }
}
